components/StateMultipleChoiceQuizPage.tsx

```diff
@@ -1,10 +1,55 @@
-import React from "react";
-import { Text, View } from "react-native";
+import React, { useEffect, useState } from "react";
+import { Text, View, Button } from "react-native";
+import { Question } from "../types/Question";
+import { quizQuestions } from "../types/Question";
+
+export default function StateMultipleChoiceQuizApp() {
+  const [currentQuestionIndex, setCurrentQuestion] = useState<number>(0);
+  const [score, setScore] = useState<number>(0);
+  const [quizFinished, setQuizFinished] = useState<boolean>(false);
+  const [shuffledQuestions, setShuffledQuestions] = useState<Question[]>([]);
+
+  useEffect(() => {
+    const shuffled = [...quizQuestions].sort(() => Math.random() - 0.5);
+    setShuffledQuestions(shuffled);
+  }, []);
```

Comment:
I don't think so this logic is working. I am getting all 14 questions while running the app.

Reply:
Its working fine when I run the app, shuffled questions too. Kindly check once again.

Reply:
The number of questions selected for the game should be lesser than the total number of questions, as per the design discussed offline.